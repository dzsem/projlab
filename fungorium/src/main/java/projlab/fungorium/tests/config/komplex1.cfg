# Konfiguráció, pályaépítés
# -------------------------

# 0 (t1)
ADD TECTON
# 1 (t2)
ADD TECTON
# 2 (t3)
ADD TECTON
# 3 (t4)
ADD THREADKILLINGTECTON

REGISTER NEIGHBOUR 0 1
REGISTER NEIGHBOUR 0 3
REGISTER NEIGHBOUR 1 2

# 4
ADD MUSHROOMBODY 0 0
# 5
ADD MUSHROOMTHREAD 0 0

# 6
ADD INSECT 1 2

SET MUSHROOMTHREAD STATE 5 UNCUT GROWN
SET TECTONSPLITCHANCE 0
SET TECTONKILLCHANCE 100
