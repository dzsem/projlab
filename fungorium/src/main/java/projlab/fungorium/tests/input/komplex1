# Komplex teszt 1
# ------------------------------------------
LOAD config/komplex.cfg

# Input
# ------------------------------------------
# 7
MUSHROOMTHREAD 5 GROW 1
# 8
MUSHROOMTHREAD 5 GROW 3

NEXTROUND
NEXTROUND
NEXTROUND

# 9
MUSHROOMTHREAD 7 GROW 2

NEXTROUND
NEXTROUND
NEXTROUND

INSECT 6 MOVE 1
# Print states
# ------------------------------------------
SAVE komplex1.out