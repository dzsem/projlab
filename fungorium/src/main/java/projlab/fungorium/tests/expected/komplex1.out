TECTON 0 2 4 1 0
TECTON 1 2 -1 1 0
TECTON 2 1 -1 1 0
THREADKILLINGTECTON 3 1 -1 1 0
MUSHROOMTHREAD 5 4 1 UNCUT GROWN
MUSHROOMTHREAD 7 -1 2 UNCUT GROWN
MUSHROOMTHREAD 9 -1 1 UNCUT GROWN
INSECT 6 1 1 1 0